_security/configuration/index.md

```diff
@@ -13,16 +13,96 @@ redirect_from:
 
 The plugin includes demo certificates so that you can get up and running quickly. To use OpenSearch in a production environment, you must configure it manually:
 
-1. [Replace the demo certificates]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/docker/#configuring-basic-security-settings).
-1. [Reconfigure `opensearch.yml` to use your certificates]({{site.url}}{{site.baseurl}}/security/configuration/tls).
-1. [Reconfigure `config.yml` to use your authentication backend]({{site.url}}{{site.baseurl}}/security/configuration/configuration/) (if you don't plan to use the internal user database).
-1. [Modify the configuration YAML files]({{site.url}}{{site.baseurl}}/security/configuration/yaml).
-1. If you plan to use the internal user database, [set a password policy in `opensearch.yml`]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#opensearchyml).
-1. [Apply changes using the `securityadmin` script]({{site.url}}{{site.baseurl}}/security/configuration/security-admin).
-1. Start OpenSearch.
-1. [Add users, roles, role mappings, and tenants]({{site.url}}{{site.baseurl}}/security/access-control/index/).
-
-If you don't want to use the plugin, see [Disable security]({{site.url}}{{site.baseurl}}/security/configuration/disable-enable-security/).
+### [Replace the demo certificates]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/docker/#configuring-basic-security-settings)
```

Comment:
Global: Are we OK with hyperlinked headings?

Reply:
I'm okay with them personally. Open to another take though.

Reply:
To clarify, I'm questioning both the utility and the style choice. Why are we linking the reader to different pages if the content relating to the headings is provided underneath them, as expected? If we have a good reason for doing so, then this should be explained to the reader so that it's clear to them for what reason they would need to select the headings to navigate to additional information as opposed to simply reading the current page. It would also be an inconsistent style choice, as I don't know that we do this elsewhere.

Reply:
Yes I see the point Nate, given it is quite inconsistent, on second thoughts I'll take it out and add any links as per the usual way. Thanks

Reply:
This is addressed and all links are now added as per the standard approach - thank you.